packages/environment/package.json

```diff
@@ -28,8 +28,10 @@
   },
   "devDependencies": {
     "@enzymefinance/sdk": "workspace:*",
+    "dotenv-cli": "^7.4.2",
```

Comment:
why do we add those two libs? I don't see usage

Reply:
Removed.